# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1696185960 -7200
# Parent  1924340e6bda40d08fcfa75b60d4c80b360e10c2
Feature: Linux system tray, add Mate to supported desktops.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -1342,17 +1342,17 @@ pref("toolkit.telemetry.bhrPing.enabled"
 #else
   pref("toolkit.telemetry.ecosystemtelemetry.enabled", false);
 #endif
 
 #if defined(XP_WIN) || defined(UNIX_BUT_NOT_MAC)
 pref("mail.minimizeToTray", false);
 #endif
 #ifdef UNIX_BUT_NOT_MAC
-pref("mail.minimizeToTray.supportedDesktops", "kde,gnome,xfce");
+pref("mail.minimizeToTray.supportedDesktops", "kde,gnome,xfce,mate");
 #endif
 
 pref("prompts.defaultModalType", 3);
 pref("prompts.contentPromptSubDialog", false);
 
 // The URL for the privacy policy related to recommended extensions.
 pref("extensions.recommendations.privacyPolicyUrl", "https://www.mozilla.org/en-US/privacy/thunderbird/#addons");
 
